physical/consul.go

```diff
@@ -513,7 +513,13 @@ shutdown:
 // checkID returns the ID used for a Consul Check.  Assume at least a read
 // lock is held.
 func (c *ConsulBackend) checkID() string {
-	return "vault-sealed-check"
+	return fmt.Sprintf("vault-sealed-check:%s:%d", c.advertiseHost, c.advertisePort)
```

Comment:
With this name change (which you unfortunately need) is there a chance you'll end up with two checks when you migrate? I suppose if we properly deregister then this isn't an issue.